pkg/controller/directvolumemigration/rsync.go

```diff
@@ -1261,6 +1262,8 @@ func (t *Task) hasAllProgressReportingCompleted() (bool, error) {
 						pendingSinceTimeLimitPods = append(pendingSinceTimeLimitPods, fmt.Sprintf("%s/%s", podProgress.Namespace, podProgress.Name))
 					}
 				}
+			case dvmp.Status.PodPhase == "":
```

Comment:
This is not needed if we fix the wrong nil pointer down below right?

Reply:
not sure, when you see `dvmp.Status.PodPhase == ""` what is the right thing to do?

Reply:
no worries. we will just let it be here I guess for safety. cant think of a scenario atm. doesnt hurt to keep